src/mscorlib/shared/System/InvalidTimeZoneException.cs

```diff
@@ -6,6 +6,8 @@
 
 namespace System
 {
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
In desktop, this is already type forwarded: ` [TypeForwardedFrom("System.Core, Version=3.5.0.0, ..`. I assume that means that we won't be able to deserialize one of these if it was serialized on 3.5 or before, as it would appear to be in System.Core ?

Reply:
good catch, I must have missed that one. That simply means that we need to put the same TypeForwardedFrom attribute here in Core then everything should work fine :)

Reply:
So that the System.Core shim entry for the InvalidTimeZoneException gets gated to System.Private.Corelib.dll